run_configs/build_all.py

```diff
@@ -4,20 +4,33 @@
 #  For further details please refer to the file COPYRIGHT
 #  which you should have received as part of this distribution
 # ##############################################################################
+
 import os
 from pathlib import Path
 
-from fab.steps.compile_fortran import get_fortran_compiler
-from fab.tools import run_command
+from fab.tools import Categories, Tool, ToolBox
+
+
+class Script(Tool):
+    '''A simple wrapper that runs a shell script.
+    :name: the path to the script to run.
+    '''
+    def __init__(self, name: Path):
+        super().__init__(name=name.name, exec_name=str(name),
+                         category=Categories.MISC)
+
+    def check_available(self):
+        return True
 
 
 # todo: run the exes, check the output
 def build_all():
 
+    tool_box = ToolBox()
+    compiler = tool_box[Categories.FORTRAN_COMPILER]
```

Comment:
I would tend to agree (and am happy to read your opinion :) ). I expect a lot of the examples to be changed later (e.g. our build scripts for LFRic look quite different from what is currently in run_configs), but since we are changing quite a lot, my idea is that we update all run_config once we done with the changes (e.g. better compiler-specific flags, and handling of linker). We will maintain the scripts so that they kind of work the way they worked originally till then.